### DISCUSSION TITLE
How to make CrewAI use external DB for multi-user systems?

### DISCUSSION THREAD

**Discussion by BarryBahrami** [crewAIInc/crewAI#1090] [Q&A]
How could I have CrewAI use an external choma, etc, so that I can make a multi user system all sharing this same information?

I have looked through the docs and I see no way to set that.

TYIA

**Comment by theCyberTech** [comment]
you can jsut import it as normal and setup in your code I.E

import chromadb
chroma_client = chromadb.Client()

https://docs.trychroma.com/getting-started
